backend/apps/events/views.py

```
@@ -34,49 +37,71 @@ def _parse_and_validate_times(start_time_str, end_time_str, validate_future=Fals
-        
+
         if host_membership:
             ChatGroupMember.objects.create(
                 group=chat_group,
                 membership=host_membership,
                 is_admin=True,
             )
-        
+
         return chat_group, None
     except Exception as e:
```

Ruff (0.15.9)

[warning] Do not catch blind exception: `Exception`

(BLE001)